src/hermes_claude_agent_sdk/tool_bridge.py

```
@@ -0,0 +1,815 @@
+                    name,
+                    "Host tool execution failed",
+                    True,
+                )
+            host_result = await result
+        except asyncio.CancelledError:
+            return ToolCallResult(
+                safe_request_id,
+                self._correlation_id,
+                name,
+                "Tool call cancelled",
+                True,
```

Ruff (0.16.2)

[warning] Boolean positional value in function call

(FBT003)